spec/requests/user_actions_controller_spec.rb

```diff
@@ -169,4 +169,15 @@ def list_and_check(action_type, expected_response)
       end
     end
   end
+
+  describe '#private_messages' do
```

Comment:
I don't think this is the right spot because the route you're testing below now relies on `ListController#private_messages_tag` and not the `UserActionsController`.